deepmd/entrypoints/main.py

```diff
@@ -205,6 +209,13 @@ def parse_args(args: Optional[List[str]] = None):
         default=None,
         help="the frozen nodes, if not set, determined from the model type",
     )
+    parser_frz.add_argument(
+        "-w",
+        "--nvnmd-weight",
+        type=str,
+        default=None,
+        help="the frozen weights, if set, save the weights of model into the file nvnmd/weight.npy",
```

Comment:
Setting to any string will save to file nvnmd/weight.npy? It is not a good design